internal/fs/fs_test.go

```diff
@@ -752,9 +752,6 @@ func TestIsDir(t *testing.T) {
 			if want.exists != got {
 				t.Fatalf("expected %t for %s, got %t", want.exists, f, got)
 			}
-			if !want.err {
-				t.Fatalf("expected no error, got %v", err)
-			}
 		}
 
 		if got != want.exists {
```

Comment:
Why did you remove the check for the return value of `IsDir`?

Anyway, I would rewrite the for loop as follows since it's checking for the same thing twice:

```go
for f, want := range tests {
	got, err := IsDir(f)
	if err != nil && !want.err {
		t.Fatalf("expected no error, got %v", err)
	}
	if want.exists != got {
		t.Fatalf("expected %t for %s, got %t", want.exists, f, got)
	}
}
```